simulator/func_sim/alu.h

```diff
@@ -203,6 +203,15 @@ struct ALU
     // Leading zero/ones
     template<typename I, typename T> static void clo( I* instr)  { instr->v_dst = count_leading_ones<T>( instr->v_src1); }
     template<typename I, typename T> static void clz( I* instr)  { instr->v_dst = count_leading_zeroes<T>( instr->v_src1); }
+    template<typename I> static
+    void pcnt( I* instr){
```

Comment:
Can we use that function:
https://github.com/MIPT-ILab/mipt-mips/blob/7f3a24dd1081265b95191624d5e4958f0a85b32e/simulator/infra/macro.h#L49